### DISCUSSION TITLE
Serenity shows 0 tests run in its report.

### DISCUSSION THREAD

**Discussion by DianArrieta** [serenity-bdd/serenity-core#3254] [Q&A]
When I run the tests, the report shows 0 tests run, it only works if I run the tests in parallel.
How can I run the tests sequentially? How should it be configured?

Mu runner file:

```
package starter;

import org.junit.platform.suite.api.ConfigurationParameter;
import org.junit.platform.suite.api.IncludeEngines;
import org.junit.platform.suite.api.SelectClasspathResource;
import org.junit.platform.suite.api.Suite;

import static io.cucumber.junit.platform.engine.Constants.PLUGIN_PROPERTY_NAME;

@Suite
@IncludeEngines("cucumber")
@SelectClasspathResource("/features")
@ConfigurationParameter(key = PLUGIN_PROPERTY_NAME, value = "io.cucumber.core.plugin.SerenityReporterParallel,pretty,timeline:build/test-results/timeline")
public class CucumberTestSuite {
}
```

My junit-platform.properties file:

```
cucumber.execution.parallel.enabled=true
cucumber.execution.parallel.config.strategy=fixed
cucumber.execution.parallel.config.fixed.parallelism=4
cucumber.execution.parallel.config.fixed.max-pool-size=4
```

I tried to set cucumber.execution.parallel.enabled=true to 'false' but it didn't work

**Comment by Rudolf-AT** [comment]
May not be related, but what worked for me at a similar issue was stopping gradle before running the tests. `./gradlew --stop`

**Comment by shashiyadavnitn** [comment]
duplicate scenario or feature description can cause this issue  .run below command to check for any duplication.Removing duplication should resolve report issue
 mvn serenity:check-gherkin
 
 to run feature in sequence you can use below config
 cucumber.execution.parallel.config.strategy=fixed
cucumber.execution.execution-mode.feature=same_thread
